packages/runtime/src/widgets/Form/Checkbox.tsx

```diff
@@ -0,0 +1,50 @@
+import { Form as AntForm } from "antd";
+import { useState } from "react";
+import { useRegisterBindings } from "framework";
+import { Checkbox } from "antd";
+import type { EnsembleWidgetProps } from "../../util/types";
+import type { FormInputProps } from "./types";
+import { WidgetRegistry } from "../../registry";
+
+export type CheckBoxProps = {
+  trailingText?: string;
+  leadingText?: string;
+} & EnsembleWidgetProps &
+  FormInputProps<boolean>;
+
+export const CheckboxWidget: React.FC<CheckBoxProps> = (props) => {
+  const [checked, setChecked] = useState(props.value || false);
+  const { values } = useRegisterBindings(
+    { ...props, value: checked },
+    props.id,
+    {
+      setValue: setChecked,
+    }
+  );
+  return (
+    <AntForm.Item
+      label={values.label}
+      name={values.label}
+      valuePropName="checked"
+      rules={[{ required: values.required }]}
+      style={{
+        margin: "0px",
+      }}
+    >
+      {" "}
```

Comment:
Is this empty space intentional?

Reply:
i guess it caused some prettier config maybe